packages/fs-detectors/src/monorepos/monorepo-managers.ts

```diff
@@ -13,13 +13,17 @@ import type { Framework } from '@vercel/frameworks';
  *   detectFramwork({ fs, frameworkList }) // returns the 'slug' field if detected, otherwise null
  *
  */
-export const monorepoManagers: Array<
-  Omit<Framework, 'description' | 'getOutputDirName'>
-> = [
+export type MonorepoManager = Omit<
+  Framework,
+  'description' | 'getOutputDirName'
+> & { darkLogo?: string };
```

Comment:
```suggestion
>
```

Reply:
Oh perfect...I didn't see that in there 💯